src/components/studies/participants/grid/ParticipantTableGrid.tsx

```diff
@@ -141,11 +141,16 @@ const EditCell: FunctionComponent<{
         return result as ParticipantEvent[]
       }
 
+      const getClientTimeZone = (column: string) => {
```

Comment:
I don't think you need a special function for getting clientTimeZone. You can modify getValString to take an optional default parameter